js/Makefile

```diff
@@ -79,6 +80,11 @@ generate: node_modules gen.sum
 .PHONY: regenerate
 regenerate: gen.clean generate
 
+.PHONY: icons.generate
+icons.generate:
+	npx icon-font-generator packages/assets/custom-icons/*.svg -o packages/assets/custom-icons/font --name custom-icons-font
```

Comment:
Here you can use something like that to check if npx is a callable program: `$(call check-program, npx)`